main.py

```diff
@@ -396,13 +400,16 @@ def build(self):
 
     def train(self, n_epoch, dataset):
         global g_args
-        train_writer = tf.summary.FileWriter(hparams.SUMMARY_DIR, g_sess.graph)
+        train_writer = tf.summary.FileWriter(hparams.SUMMARY_DIR + '/' + str(datetime.datetime.now().strftime("%m%d_%H%M%S")) + ' ' + hparams.SUMMARY_TITLE, g_sess.graph)
```

Comment:
Even it doesn't support windows right now, it's better to use `os.path.join` here.